yesod-auth/Yesod/Auth/Email.hs

```diff
@@ -709,6 +713,9 @@ setLoginLinkKey aid = do
     now <- liftIO getCurrentTime
     setSession loginLinkKey $ TS.pack $ show (toPathPiece aid, now)
 
+csrfErrorMessage :: Text
+csrfErrorMessage = "A valid CSRF token wasn't present in HTTP headers or POST parameters. Because the request could have been forged, it's been rejected altogether. Check the Yesod.Core.Handler docs of the yesod-core package for details on CSRF protection."
```

Comment:
It's a little weird because it's not an API the user would use, but I think exporting this error message from Yesod-core is preferable to pasting it here